docs/proposals/README.md

```diff
@@ -0,0 +1,27 @@
+# OpenShift-Ansible Proposal Process
+
+## Proposal Decision Tree
+TODO: Add details about when a proposal is or is not required. 
+
+## Proposal Process
+The following process should be followed when a proposal is needed:
+
+1. Create a pull request with the initial proposal
+  * Use the [proposal template][template]
+  * Name the proposal using two or three topic words with underscores as a separator (i.e. proposal_template.md)
+  * Place the proposal in the docs/proposals directory
+2. Notify the development team of the proposal and request feedback
```

Comment:
Please define 'development team' and is there a way that word can link to some kind of github generated list of said team members?